src/ray/protobuf/gcs_service.proto

```diff
@@ -583,6 +591,10 @@ message GetAllResourceUsageRequest {}
 message GetAllResourceUsageReply {
   GcsStatus status = 1;
   ResourceUsageBatchData resource_usage_data = 2;
+  /// True if gcs finds infeasible or pending actor creation tasks
+  /// locally (when gcs actor scheduler is enabled). This field is
+  /// expected to help triggering auto-scaling.
+  bool cluster_full_of_actors_detected_by_gcs = 3;
```

Comment:
hmm, is there any reason we can't reuse the ResourceUsageBatchData?

Reply:
`ResourceUsageBatchData` is a gcs proto, which is also used for table storage and pubsub (although the pub-sub one might be removed later.) So adding a `cluster_full_of_actors_detected_by_gcs` in `ResourceUsageBatchData` might increase unnecessary overhead in most cases.